docker-compose-installed.yaml

```diff
@@ -21,7 +21,11 @@ services:
       context: .
       dockerfile: ./cli/Dockerfile-installed
       args:
-        ISOLATION_ID: ${ISOLATION_ID}
+        args:
```

Comment:
Adding these proxy args should be a different commit and PR, since this change is not related to the bug fix.

Reply:
Ok, I will double check. 

I'm pretty sure our team was unable to standup the current sawtooth-seth repo without ISOLATION_ID: ${ISOLATION_ID} added to docker-compose-installed.yaml after docker-compose.yaml fixed for port 3030 connection refused in Seth issue.

Possibly due to docker 18.03.1-ce changes landing. https://docs.docker.com/release-notes/docker-ce/#stable-releases

Reply:
Please see my revised changes, tested and revised commit message.

Reply:
Compose file is invalid because you have the following structure:
```
args:
    args:
    ...
```